frontend/public/components/utils/dropdown.jsx

```diff
@@ -323,9 +313,19 @@ export class Dropdown extends DropdownMixin {
     });
 
     //Adding `dropDownClassName` specifically to use patternfly's context selector component, which expects `bootstrap-select` class on the dropdown. We can remove this additional property if that changes in upcoming patternfly versions.
-    return <div className={className} ref={this.dropdownElement} style={this.props.style}>
+    return <div className={classNames({[wrapperClassName]: wrapperClassName})} ref={this.dropdownElement} style={this.props.style}>
       <div className={classNames('dropdown', dropDownClassName)}>
-        {button}
+        {
+          noButton
+            ? <div onClick={this.toggle} className="dropdown__not-btn" id={this.props.id}>
+              <span className="dropdown__not-btn__title">{title}</span>&nbsp;<Caret />
+            </div>
+            : <button onClick={this.toggle} type="button" className={classNames('btn', 'btn-default', 'btn--dropdown', 'dropdown-toggle', {[buttonClassName]: buttonClassName})} id={this.props.id}>
```

Comment:
I think it'd be better to do `classNames('btn', 'btn--dropdown', 'dropdown-toggle', buttonClassName ? buttonClassName : 'btn-default')` so that btn-default and {buttonClassName} aren't both added.